packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -41,24 +41,30 @@ type AmountCommonProps = {
    */
   suffix?: 'decimals' | 'none' | 'humanize';
   /**
-   * Makes the prefix symbol and decimal digits small and faded
+   * Makes the currency indicator(currency symbol/code) and decimal digits small and faded
    *
    * @default true
    */
   isAffixSubtle?: true | false;
   /**
-   * Prefix to be shown before the amount value. The prefix can be either a currency symbol or a currency code.
+   * Determines the visual representation of the currency, choose between displaying the currency symbol or code.
    *
    * @default 'currency-symbol'
    */
-  prefix?: 'currency-symbol' | 'currency-code';
+  currencyIndicator?: 'currency-symbol' | 'currency-code';
   /**
    * The currency of the amount.  Note that this component
    * only displays the provided value in the specified currency, it does not perform any currency conversion.
    *
    * @default 'INR'
    * */
   currency?: Currency;
+  /**
+   * If true, the amount text will have a line through it.
+   *
+   * @default false
+   */
+  isStrikethrough?: boolean;
```

Comment:
nit: wouldn't `shouldStrikethrough` be more suitable?

Reply:
I'm fine with either `isStrikethrough` or `hasStrikethrough `.